health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/UserDownload.js

```
@@ -53,11 +53,10 @@ const UserDownload = () => {
 
     return (
         <div>
-
-
+            {isLoading && <LoaderWithGap text={t("CS_LOADING")} />}
             <Card type="secondary" >
                 <HeaderComp title="DOWNLOAD_USER_DATA" styles={{ color: "black" }} />
-                <TextBlock body="DOWNLOAD_DESC" />
+                <TextBlock body={t("DOWNLOAD_DESC")} />
                     {data?.ResourceDetails &&
                         [...data?.ResourceDetails].reverse().map((item, index) => {
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)